Resources/Private/Templates/Pages/Default.html

```diff
@@ -1,24 +1,17 @@
-<f:layout name="Default" />
+<f:layout name="Default"/>
 <f:section name="Main">
 
-  <main role="main">
+    <main role="main">
+        <f:render partial="Jumbotron" arguments="{records: content.jumbotron.records}"/>
 
-    <f:render partial="Jumbotron" arguments="{jumbotronContent: jumbotronContent}"/>
-
-    <div class="container">
-      <div class="row">
-        <div class="col-md-12">
-            <f:for each="{mainContent}" as="contentElement">
-                <f:cObject
-                    typoscriptObjectPath="tt_content.{contentElement.data.CType}"
-                    data="{contentElement.data}"
-                    table="tt_content"
-                />
-            </f:for>
+        <div class="container">
+            <div class="row">
+                <div class="col-md-12">
```

Comment:
This is probably obsolete for 1-column layouts

Reply:
Done